data/assets/default.profile

```diff
@@ -3,10 +3,10 @@
 
 #Asset
 scene/solarsystem/planets/earth/earth	required
-scene/solarsystem/planets/earth/satellites/satellites   required
+scene/solarsystem/planets/earth/satellites/satellites	required
 
 #Property
-setPropertyValue  {earth_satellites}.Renderable.Enabled false
+setPropertyValue	{earth_satellites}.Renderable.Enabled	false
```

Comment:
This is a tab-delimited file. If these are replaced with spaces, then the parsing won't work.

Reply:
They were space-delimited in the repository and I replaced them with tabs on my end